semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -258,9 +307,30 @@ def setup_parser():
     subparsers = parser.add_subparsers(dest="flavor")
     add_code_parser(subparsers)
     add_md_parser(subparsers)
+    add_compat_check_parser(subparsers)
 
     return parser
 
 
+def download_previous_version(version: str) -> str:
+    filename = f"v{version}.zip"
+    tmppath = tempfile.mkdtemp()
+    path_to_zip = os.path.join(tmppath, filename)
+    path_to_semconv = os.path.join(tmppath, f"v{version}")
+
+    semconv_vprev = (
+        f"https://github.com/open-telemetry/semantic-conventions/archive/{filename}"
```

Comment:
is it correct url? doesn't work for me.
https://github.com/open-telemetry/semantic-conventions/archive/refs/tags/v1.24.0.zip seems to working

Reply:
Thanks @lmolkova, works for me today, direct url as well, probably was some glitch